data/hai/midas.txt

```diff
@@ -0,0 +1,993 @@
+			``
+			`	Do you accept this deal from 'She of the Void', or do you want to escort the governor as planned?`
+	waypoint "Heia Due"
+	on waypoint
+		"reputation: Midas" >?= 0
+		conversation
+			`One of the unindentified ships hail you. "You did well, Captain. Here is the first part of your reward: <payment>. You may help us by taking out these Hai ships, but it may be wise not to so that the rest of the Hai government will not suspect you for this unfortunate pirate attack.`
+			`	When this is done, go to <destination> for your reward. The Unfettered are going to be in high need of food, and you should bring them as much as you can to compensate for this small... delay in Hai food supplies."`
+			`	Right after that, the governor's ship hails you, and you hear her voice, between anger and confusion. "What have you done?!" You feel like she was about to continue, but before she could all communications are jammed.`
+			`	You can see some pirates present with the Midas fleets, probably to distract other Hai ships that happen to be in the system. It might be a good idea to bribe them.`
+		payment 1000000
+	# as of yet, we cannot make the main mission fail as your escorts are boarded, meaning these will not be disables
+	npc
+		# The Alphas and their minions left Midas alone so that this would happen. They probably even gave them an anonymous tip about it.
+		government "Midas"
+		system "Heia Due"
+		personality plunders staying harvests
+		fleet "Marauders" 6
+		on provoke
+			dialog `You hear a voice the instant you fire on the ships you had pretended to ally with, seemingly coming from nowhere, "You have chosen your side. Nemo Me Impune Lacessit, <first>." You suddenly feel a strong pain in your neck.`
+			"reputation: Midas" <?= -1000
+			"midas: wronged" ++
+	npc
+		government "Pirate"
+		system "Heia Due"
+		personality entering plunders staying
+		fleet "Small Northern Pirates" 10
+	to fail
+		or
+			has "<uhai> 6 A: Loyal Escort: done"
+			has "<uhai> 6 A: Loyal Escort: aborted"
+	on fail
+		dialog `You hear a voice once alone in your quarters, "A shame, captain. Truly a shame you would not simply take an alternate path."`
+		"reputation: Midas" <?= -1000
+	to complete
+		has "<uhai> 6 A: Loyal Escort: failed"
+	on complete
+		"unfettered rep: Sookootow" -= 2
+		"reputation: Midas" <?= -1000
+		clear "unfettered: stopped on Greenbloom"
+		log `The plan was to escort a Hai fleet carying supplies and passengers into Unfettered space but an opportunity to feed them to the wolves instead was presented, and taken.`
+		payment 4000000
+		dialog
+			`As before, a voice addresses you once you are alone in your quarters. "We have transferred <payment> to your account, and made sure the Chief will not make your life difficult.`
+			`	"Do be aware, however, that a food situation has arrisen, not because of our small operation, but because of the bomb's presence in the first place, placed by our rivals.`
```

Comment:
```suggestion
			`	"Do be aware, however, that a food situation has arisen, not because of our small operation, but because of the bomb's presence in the first place, placed by our rivals.`
```